tools/patch-onchainid.sh

```
@@ -0,0 +1,429 @@
+            fi
+        fi
+
+        FILES_PROCESSED=$((FILES_PROCESSED + 1))
+    done
+
+    log_success "Pragma statement updates completed"
+}
+
+update_import_statements() {
+    local onchainid_dir="$1"
+    local os_type="$(get_os_type)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)